### ISSUE TITLE
Mark project settings in the license page and the excel Export

### ISSUE THREAD

**Issue by Janpopan** [porscheinformatik/sonarqube-licensecheck#107]

**Comment by derhecht** [comment]
could you may explain/scribble your wish a bit?

**Comment by Janpopan** [comment]
Yes, sometimes you have to know, if this licence is allowed globaly or only in this project. 
And so it would be great to have this Information in the report or Excel export

**Comment by derhecht** [comment]
@derkoe I'm quite new to Sonar plugins and this project as well. Which function is called when triggering /api/measures/search?projectKeys=${this.component.key}&metricKeys=licensecheck.license,licensecheck.dependency from dashboard to generate Excel?
My idea is to just add a p/g in front of the license status string here and resolve it in the JS logic. As I think extend the model for this feature might be a bit to much.
